app/models/spree/reimbursement_decorator.rb

```diff
@@ -3,6 +3,10 @@ module ReimbursementDecorator
     def pos_refunded!
       update_attribute(:pos_refunded, true)
     end
+
+    def try_set_order_to_return!
```

Comment:
Nitpicking a bit: I'm not a big fan of the `try_` prefix here, for the same reason I don't like the `Object#try` method: you're not _trying_ to trigger the return event, you _are_, and "try" evokes the "try/catch/finally" structure, and you're not rescuing the exception.

Long-winded way to say, I'd name this simply `return_order!` 😄